### ISSUE TITLE
Upgrade vue-eslint-parser to 6.0.0 cause valid-v-bind false negative

### ISSUE THREAD

**Issue by xcatliu** [vuejs/eslint-plugin-vue#898]
**Tell us about your environment**	
* **ESLint version:** 5.16.0
* **eslint-plugin-vue version:** 5.2.2
* **Node version:**	10.13.0

**Please show your full configuration:**

```js
module.exports = {
  parser: 'vue-eslint-parser',
  plugins: ['vue'],
  rules: {
    // override/add rules settings here, such as:
    // 'vue/no-unused-vars': 'error'
    'vue/valid-v-bind': 'error'
  }
}
```

**What did you do?**	

```vue
<!-- bad 语法错误的 v-bind -->
<template>
  <div>
    <div v-bind />
    <div :aaa />
    <div :aaa.bbb="foo" />
  </div>
</template>
```

**What did you expect to happen?**
I expect to have errors after upgrade vue-eslint-parser to 6.0.0

**What actually happened?**

No errors

I created a repo for this issue: https://github.com/xcatliu/test-eslint-vue

**Comment by xcatliu** [comment]
Related issue in vue-eslint-parser https://github.com/mysticatea/vue-eslint-parser/issues/46

**Comment by ota-meshi** [comment]
Thank you for this issue.

`eslint-plugin-vue@5` and `vue-eslint-parser@6` are not compatible. Please use `eslint-plugin-vue@6`.
